engine/jobmaster/dm/metadata/info.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metadata
+
+import (
+	"context"
+
+	"github.com/coreos/go-semver/semver"
+	frameModel "github.com/pingcap/tiflow/engine/framework/model"
+	"github.com/pingcap/tiflow/engine/pkg/adapter"
+	metaModel "github.com/pingcap/tiflow/engine/pkg/meta/model"
+)
+
+// Info represents the cluster info.
+type Info struct {
+	State
+
+	Version semver.Version
+}
+
+// NewInfo creates a new Info instance.
+func NewInfo(version semver.Version) *Info {
+	return &Info{
+		Version: version,
+	}
+}
+
+// InfoStore manages the state of info.
+type InfoStore struct {
+	*TomlStore
+
+	id frameModel.MasterID
```

Comment:
Do we still need ID? now kvclient should provide job-level isolation

Reply:
cc @maxshuang , Do we still need to store metadata with job prefix?